.gitignore

```diff
@@ -1,18 +1,16 @@
-*.log
-*.pot
-*.pyc
-__pycache__
-.vagrant
-build
-ENV
-.coverage
-htmlcov
+# Consider using a global git ignore file for files and directories that
+# are commonly generated by your OS, IDE, Python, development tools etc.
+# A fairly extensive starting point would be:
+#
+# https://www.gitignore.io/api/windows,linux,osx,python,vagrant,sass,idea,visualstudiocode
+#
+# Store the content of this file in your home directory as ~/.gitignore_global
+# and configure git using the following command:
+#
+# $ git config --global core.excludesfile ~/.gitignore_global
 
-static-root
-.sass-cache
```

Comment:
`.sass-cache` is an implementation detail of the Ruby implementation of sass. I'd guess many Python projects don't depend on the Ruby gem (personally python.org is the only one I've worked on) and use libsass-python instead. It would be convenient if we keep `.sass-cache` for now.

Reply:
It's included in the list that's linked in the comment (part of the sass set). It's fairly common in older projects that use compass.

I'll add it back and remove the sass set from the global gitignore example link.